packages/agent/test/support/servers.ts

```diff
@@ -0,0 +1,72 @@
+import * as http from 'http'
+import * as https from 'https'
+import * as fs from 'fs'
+import * as os from 'os'
+import * as path from 'path'
+import * as express from 'express'
+import * as Promise from 'bluebird'
+import { CA } from '@packages/https-proxy'
+import * as Io from '@packages/socket'
+
+function createExpressApp() {
+  const app = express()
+
+  app.get('/get', (req, res) => {
+    res.send('It worked!')
+  })
+
+  app.get('/empty-response', (req, res) => {
+    // ERR_EMPTY_RESPONSE in Chrome
+    setTimeout(() => res.connection.destroy(), 100)
+  })
+
+  return app
+}
+
+function getLocalhostCertKeys() {
+  return CA.create(fs.mkdtempSync(path.join(os.tmpdir(), 'cy-test-')))
```

Comment:
move the `fs.mkdtempSync` call into `CA.create(...)` and utilize `fs-extra` to `ensureDirAsync` or something similar.